src/plugins/orangehrmCoreOAuthPlugin/Controller/OAuthClientController.php

```diff
@@ -32,8 +33,7 @@ class OAuthClientController extends AbstractVueController
     public function preRender(Request $request): void
     {
         $component = new Component('oauth-client-list');
-        // TODO move the hard coded mobile client id to some central place
-        $component->addProp(new Prop('unselectable-client-ids', Prop::TYPE_ARRAY, ['orangehrm_mobile_app']));
+        $component->addProp(new Prop('unselectable-client-ids', Prop::TYPE_ARRAY, [OAuthService::PUBLIC_MOBILE_CLIENT_ID]));
```

Comment:
are we keeping mobile app client ID the same or changing to match format of other client ids?

Reply:
have to keep same, because we are going to hardcode it in the mobile app side